<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mirrorDlg</class>
 <widget class="QDialog" name="mirrorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mirror configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QCheckBox" name="flipH">
     <property name="text">
      <string> Flip Interferogram left to right on loading </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="nullCB">
     <property name="text">
      <string>Artificial Null</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Interferogram
Wave Length nm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="z8">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lambda"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="ReadBtn">
     <property name="text">
      <string>Read Existing File</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Save in a File</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Fringe Spacing in waves:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fringeSpacingEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set to 1 if using a Bath interferometer or similar.  Set to .5 if testing in autocollimation with same.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Elliptical outline</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="ellipseShape">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Elliptical FLat or Nulled Optics</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Vertical Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minorAxisEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mirror </string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QLabel" name="rocLab">
        <property name="text">
         <string>Roc:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="FNumber"/>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Obstruction:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="fnumberLab">
        <property name="text">
         <string>FNumber:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLineEdit" name="diameter">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Daimeter of mirror or Horizontal axis of ellipse.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="roc">
        <property name="toolTip">
         <string>Radius of curvature of mirror or 0 if testing a flat</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="2">
       <widget class="QLineEdit" name="obs"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Diameter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QCheckBox" name="unitsCB">
        <property name="text">
         <string>Units in mm</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Conic Constant:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QLineEdit" name="cc">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set to 0 if testing a flat or a sphere.&lt;/p&gt;&lt;p&gt;Set to -1 if testing a parabaloid.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mirrorDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mirrorDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
